config/config_test.go

```diff
@@ -13,8 +13,9 @@ func TestGetReturnsDefaultValues(t *testing.T) {
 		So(configuration.BindAddr, ShouldEqual, ":21800")
 		So(configuration.DatabakerImportTopic, ShouldEqual, "data-bake-job-available")
 		So(configuration.SecretKey, ShouldEqual, "0C30662F-6CF6-43B0-A96A-954772267FF5")
-		So(configuration.ServiceAuthToken, ShouldEqual, "0C30662F-6CF6-43B0-A96A-954772267FF5")
+		So(configuration.ServiceAuthToken, ShouldEqual, "Bearer 0C30662F-6CF6-43B0-A96A-954772267FF5")
```

Comment:
I don't think Bearer should be included here, it should be being set by the client

Reply:
good spot